helper-cli/src/main/kotlin/commands/ScanStorageCommand.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (C) 2021 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.helper.commands
+
+import com.github.ajalt.clikt.core.CliktCommand
+import com.github.ajalt.clikt.core.subcommands
+
+import org.ossreviewtoolkit.helper.commands.scanstorage.DeleteCommand
+
+class ScanStorageCommand : CliktCommand(
```

Comment:
> What's the reason for adding a subcommand?

Because I can see multiple commands regarding the scan-results. Like the `ListStoredScanResultsCommand` that would also fit this subcommand. E.g. as `read`.

> Listing and deleting are very similar because in terms of SQL the only difference is "DELETE FROM" vs "SELECT FROM", thus all other logic would be redundant.

While that is correct, the `ListStoredScanResultsCommand` also works for other storage types. The `DeleteCommand` only works for the postgres storage, because a delete functionality is not yet implemented in the production code, and the way I implemented it here is IMO too specific to reuse it in any meaningful way.

What is your suggested change?

Reply:
> What is your suggested change?

No change suggested. I just wasn't able to judge whether a subcommand makes sense based on the information provided. Which is why I asked.